src/app/services/user.service.ts

```diff
@@ -1,10 +1,13 @@
 import { Injectable } from '@angular/core';
 import { HttpHeaders, HttpClient } from '@angular/common/http';
 import { Observable } from 'rxjs';
+import { User } from '../models/user';
 const httpOptions = {
   headers: new HttpHeaders({ 'Content-Type': 'application/json',
 'Fineract-Platform-TenantId': 'mobile',
-Authorization: 'Basic bWlmb3M6cGFzc3dvcmQ=' })
+Authorization: 'Basic bWlmb3M6cGFzc3dvcmQ=',
```

Comment:
@CajetanRodrigues hardcoded.

Reply:
Ok I will resolve that

Reply:
> @CajetanRodrigues Login using a recently registered user is not working.

I have resolved that. You can now sign up and login the same credentials you put in the form. Although the authentication flow is temporary, we could fix it in the near future. For now this PR needs to get merged to have the base fixed once and for all. So that I as well as upcoming contributors can create new components on top of the base.